src/pages/CatEdit.test.js

```diff
@@ -0,0 +1,24 @@
+// import App from "./App";
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import CatEdit from "./pages/CatEdit";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+fdescribe("When Cat Edit renders", () => {
+  let renderedCatEdit;
+  beforeEach(() => {
+    // arrange
+    renderedCatEdit = shallow(<CatEdit />);
+  });
+
+  it("displays Cat Edit page", () => {
+    // act
+    // user is looking for cat edit page
+    const renderedEdit = renderedCatEdit.find("CatEdit");
+
+    // assert
+    expect(renderedEdit.length).toEqual(1);
+  });
```

Comment:
```suggestion
  it("displays a heading", () => {
    // act
    // user is looking for cat edit page
    const heading = renderedCatEdit.find("h1");

    // assert
    expect(heading.text()).toEqual('This is a CatEdit');
  });
```
this test should not be looking for the component itself rather an element or component within CatEdit.